### ISSUE TITLE
Lowercase epsilon symbol not being read by JAWS (when MathCAT is not enabled)

### ISSUE THREAD

**Issue by jenmarsala** [daisy/math-a11y#42]
<h1>Description</h1>
When creating math content using Word's Equation Editor, JAWS does not read the lowercase epsilon symbol created with the \epsilon autocorrect code when reading with JAWS'  native math support, i.e., MathCAT is NOT enabled. (Note that JAWS does, however, read the epsilon symbol when the \varepsilon autocorrect code is used.) The epsilon symbol is used in mathematics to represent a small quantity; students often first see it in Calculus in epsilon-delta proofs when learning about limits.

<h2>Expected Behavior</h2>
When typing \epsilon followed by spacebar in Word's equation editor, JAWS should read, "math content epsilon."<br/>

<h2>Actual Behavior</h2>
When typing \epsilon followed by spacebar in Word's equation editor, JAWS reads "math content" but does not read the epsilon symbol. (The symbol does show up correctly visually.)<br/><br/>

Note that when typing \varepsilon followed by spacebar, JAWS DOES correctly read, "math content epsilon." (The \varepsilon symbol is a slightly different visual symbol.)<br/>

<h1>Tested Version Information</h1>

<h2>Windows OS</h2>
OS Version:<br/>
-- OS Name: Microsoft Windows 10 Enterprise<br/>
-- Version: 10.0.19045 Build 19045 <br/>
-- Locale: United States<br/><br/>
Microsoft Word version:<br/>
-- Microsoft 365 Apps for Enterprise<br/>
-- Version 2410 (Build 18129.20158 Click-to-Run)<br/><br/>
JAWS Version:<br/>
-- JAWS 2025.2410.106<br/>

**Comment by jenmarsala** [comment]
I concur that this is no longer an issue in the default settings of JAWS 2025, since MathCAT is now enabled by default (rather than JAWS' previous native math reader). See notes below version information.

Tested in the following environment:
- Windows 10 Enterprise (Version 10.0.19045 Build 19045)
- Microsoft 365 Apps for Enterprise, Version 2508 (Build 19127.20240 Click-to-Run)
- JAWS 2025.2028.120

Do we need to do any of the following before closing the issue?
- Disable MathCAT in JAWS 2025 to test this issue again (if this is even possible - I don't see an option to do so)
- Test the issue on previous versions of JAWS (2024, 2023, etc.) for people who may not have an updated version of JAWS. Since the cost of the software can be substantial, some people may still be using an older version.

**Comment by brichwin** [comment]
@jenmarsala: 
- I'm not aware of a means to disable MathCAT in JAWS 2025 (happily)
- I doubt there is a reason to test in prior JAWS versions as Freedom does not release fixes for prior versions (as far as I know) and this issue was likely with how JAWS behaved in prior versions.
